src/main/scala/mesosphere/marathon/api/v2/AppHelpers.scala

```diff
@@ -6,7 +6,7 @@ import mesosphere.marathon.api.{Rejection, RejectionException}
 import mesosphere.marathon.api.v2.Validation._
 import mesosphere.marathon.api.v2.validation.AppValidation
 import mesosphere.marathon.core.appinfo.{AppSelector, Selector}
-import mesosphere.marathon.plugin.auth.{AuthorizedAction, Authorizer, CreateRunSpec, Identity, UpdateRunSpec, ViewRunSpec}
+import mesosphere.marathon.plugin.auth._
```

Comment:
People let this one through in the original PR 🙈

Reply:
@zen-dog I commented on it and it was merged anyways :( 

https://github.com/mesosphere/marathon/pull/6651/files#r232423958

Reply:
Oh well

Reply:
hmm, that's not cool. I remember changing that. I'll check my intellij settings, maybe it was automatically optimised.